sdk/core/azure-core/src/context.cpp

```diff
@@ -5,11 +5,7 @@
 
 using namespace Azure::Core;
 
-Context& Azure::Core::Context::GetApplicationContext()
-{
-  static Context context;
-  return context;
-}
+Context Context::ApplicationContext;
```

Comment:
Do we need this line here or can it be removed?

Reply:
Static data members need this. It is allocating the place that actually stores the variable. The one you put in the class is just a declaration, and without this, should you use static variable, you get linker error.